nautobot/docs/overview/index.md

```diff
@@ -0,0 +1,160 @@
+
+3. **Platform for Network Automation Apps** The Nautobot App system empowers users to develop custom Network Automation Apps tailored to their specific needs. Whether lightweight or complex, these apps benefit from Nautobot's existing features such as authentication, permissions, webhooks, GraphQL, and change logging, significantly reducing development time by up to 70%. This integration not only accelerates the development process but also leverages the extensive data already stored in Nautobot, providing a powerful and efficient platform for building network automation solutions.
+
+## Getting Started
+
+<!-- markdownlint-disable no-inline-html -->
+<div class="grid cards" markdown>
+
+- :material-cloud-download:{ .lg .middle } **Installing Nautobot**
+
+    ---
+    Get up and running with a [Nautobot install](../user-guide/administration/installation/index.md) on your own Linux VM or in a Docker environment.
+
+- :material-cog:{ .lg .middle } **Configuring Nautobot**
+
+    ---
+    Learn about the many [configuration options](../user-guide/administration/configuration/index.md) that Nautobot offers for fine-tuning your installation.
+
+- :material-play-network:{ .lg .middle } **Getting Started with Nautobot**
+
+    ---
+    Dive into [how to use Nautobot](../user-guide/feature-guides/getting-started/index.md) and the key components of the core web interface.
+    Learn how [Nautobot Apps](../apps/index.md) can expand Nautobot's functionality.
+
+    ---
+
+- :material-api:{ .lg .middle } **Nautobot APIs!**
+
+    ---
+    Dive into the [REST](../user-guide/platform-functionality/rest-api/overview.md) and [GraphQL](../user-guide/platform-functionality/graphql.md) APIs.
+
+- :material-language-python:{ .lg .middle } **Nautobot SDKs**
+
+    ---
+    Nautobot has a [Python SDK](https://pynautobot.readthedocs.io/) and [Ansible modules](https://galaxy.ansible.com/ui/repo/published/networktocode/nautobot/docs/) to interact with Nautobot in a programmatic way.
+
+</div>
+<!-- markdownlint-enable no-inline-html -->
+
+## Nautobot Apps
+
+Nautobot has a thriving ecosystem of **Apps**, developed as separate projects, for which you can find links to documentation under the [Nautobot Apps](../apps/index.md) section.
+
+<!-- markdownlint-disable no-inline-html -->
+<div class="grid cards" markdown>
+
+- ![Golden Config](../assets/app-icons/icon-GoldenConfiguration.png){style="height: 35px; margin-bottom: 0em" .middle } **Golden Configuration**
+
+    ---
+    [Golden Configuration](https://github.com/nautobot/nautobot-app-golden-config) backs up configurations, generates intended state configurations, compares them for compliance and remediates device configurations.
```

Comment:
Feels like these links should point to the docs for each app instead of to their github repos?